src/css/scheme/dark.scss

```
@@ -0,0 +1,58 @@
+:root.html--dark {
+    // dark theme palette
+    $night-1  : #1f2d3b;
+    $night-2  : #283850;
+    $night-3  : #3a4958;
+    $night-4  : #495b6d;
+    $night-5  : #546a80;
+    $night-9  : #90a5bd;
+    $night-b  : #bdcbdb;
+    $night-c  : #cedef1;
+    $night-d  : #d9e3ee;
+    $night-e  : #e6eff8;
+    $primary-1: #3273dc;
+    $primary-2: #5596ff;
+    
+    // overriding variables with dark theme colors
+    
+    --background                        : #{$night-1};
```

Not a fan of this formatting, I'd prefer it to be:
```suggestion
    --background: #{$night-1};
```

---

Ah, gotcha. Yeah, the main reason I did that was so that it's easier to do multi-cursor copying of names and values and such, but that's probably not a big priority. I can remove all that extra whitespace for sure